assets/sass/all-digital/base/_defaults.scss

```diff
@@ -28,7 +28,8 @@ body {
   color: map-get($ui-blue-map, text-color);
 
   @include respond-to(700) {
-    background-image: url('data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%3E%3Cdefs%3E%3Cfilter%20id%3D%22b%22%20color-interpolation-filters%3D%22sRGB%22%3E%3CfeGaussianBlur%20stdDeviation%3D%2255%22%2F%3E%3C%2Ffilter%3E%3ClinearGradient%20id%3D%22a%22%20gradientUnits%3D%22userSpaceOnUse%22%20x1%3D%22550%22%20y1%3D%224%22%20x2%3D%22550%22%20y2%3D%22343%22%3E%3Cstop%20offset%3D%220%22%20stop-color%3D%22%230066A0%22%2F%3E%3Cstop%20offset%3D%22.5%22%20stop-color%3D%22%23076699%22%2F%3E%3Cstop%20offset%3D%221%22%20stop-color%3D%22%23003A74%22%2F%3E%3C%2FlinearGradient%3E%3C%2Fdefs%3E%3Cpath%20fill%3D%22url(%23a)%22%20d%3D%22M0%200h1100v343H0z%22%2F%3E%3Cg%20filter%3D%22url(%23b)%22%3E%3Cg%20fill%3D%22%230E74A0%22%3E%3Cellipse%20cx%3D%2272%22%20cy%3D%22121%22%20rx%3D%2295%22%20ry%3D%2249%22%2F%3E%3Cellipse%20cx%3D%22781%22%20cy%3D%22171%22%20rx%3D%22132%22%20ry%3D%22166%22%2F%3E%3Cellipse%20cx%3D%221041%22%20cy%3D%22231%22%20rx%3D%22135%22%20ry%3D%22160%22%2F%3E%3Cellipse%20cx%3D%22541%22%20cy%3D%22195%22%20rx%3D%22132%22%20ry%3D%22125%22%2F%3E%3C%2Fg%3E%3Cg%20fill%3D%22%230057A0%22%3E%3Cellipse%20cx%3D%22383%22%20cy%3D%2223%22%20rx%3D%22205%22%20ry%3D%22136%22%2F%3E%3Cellipse%20cx%3D%22278%22%20cy%3D%22270%22%20rx%3D%2283%22%20ry%3D%2232%22%2F%3E%3Cellipse%20cx%3D%221083%22%20cy%3D%2223%22%20rx%3D%22117%22%20ry%3D%2258%22%2F%3E%3C%2Fg%3E%3C%2Fg%3E%3C%2Fsvg%3E');
+    background-color: linear-gradient(to bottom,  #0066a0 0%,#076699 50%,#003a74 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
```

Comment:
I'm pretty sure gradients are only supported on the background shorthand or background-image rules

Reply:
also the laying for background images, the "image" you want on top goes first, so:
```
background-image: url('data:image/svg+xml;charset=UTF-8,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20preserveAspectRatio%3D%22none%22%3E%0A%20%20%3Cdefs%3E%0A%20%20%20%20%3Cfilter%20id%3D%22b%22%20color-interpolation-filters%3D%22sRGB%22%3E%0A%20%20%20%20%20%20%3CfeGaussianBlur%20stdDeviation%3D%2255%22%2F%3E%0A%20%20%20%20%3C%2Ffilter%3E%0A%20%20%20%20%3ClinearGradient%20id%3D%22a%22%20gradientUnits%3D%22userSpaceOnUse%22%20x1%3D%22550%22%20y1%3D%224%22%20x2%3D%22550%22%20y2%3D%22343%22%3E%0A%20%20%20%20%20%20%3Cstop%20offset%3D%220%22%20stop-color%3D%22%230066A0%22%2F%3E%0A%20%20%20%20%20%20%3Cstop%20offset%3D%22.5%22%20stop-color%3D%22%23076699%22%2F%3E%0A%20%20%20%20%20%20%3Cstop%20offset%3D%221%22%20stop-color%3D%22%23003A74%22%2F%3E%0A%20%20%20%20%3C%2FlinearGradient%3E%0A%20%20%3C%2Fdefs%3E%0A%20%20%3Cg%20filter%3D%22url%28%23b%29%22%3E%0A%20%20%20%20%3Cg%20fill%3D%22%230E74A0%22%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%2272%22%20cy%3D%22121%22%20rx%3D%2295%22%20ry%3D%2249%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22781%22%20cy%3D%22171%22%20rx%3D%22132%22%20ry%3D%22166%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%221041%22%20cy%3D%22231%22%20rx%3D%22135%22%20ry%3D%22280%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22541%22%20cy%3D%22195%22%20rx%3D%22132%22%20ry%3D%22125%22%2F%3E%0A%20%20%20%20%3C%2Fg%3E%0A%20%20%20%20%3Cg%20fill%3D%22%230057A0%22%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22383%22%20cy%3D%2223%22%20rx%3D%22205%22%20ry%3D%22136%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%22278%22%20cy%3D%22270%22%20rx%3D%2283%22%20ry%3D%2232%22%2F%3E%0A%20%20%20%20%20%20%3Cellipse%20cx%3D%221083%22%20cy%3D%2223%22%20rx%3D%22117%22%20ry%3D%2258%22%2F%3E%0A%20%20%20%20%3C%2Fg%3E%0A%20%20%3C%2Fg%3E%0A%3C%2Fsvg%3E'),
    linear-gradient(to bottom, #0066a0 0%, #076699 50%, #003a74 100%);
```